enterprise/setup/index.mdx

```diff
@@ -48,29 +51,32 @@ docker pull quay.io/pganalyze/enterprise:[version]
 Please note that `[version]` should be replaced with the version you are using. We recommend using [the latest release](/docs/enterprise/releases).
 
 
-### Step 2: Setting Environment Variables
+### Step 2: Setting environment variables
 
 The Docker image uses environment variables that are passed in to determine how to access outside services, like PostgreSQL, LDAP and mail servers.
 
-Here is an example .env file to get you started:
+We recommend keeping the environment variables in the file `/etc/pganalyze.env`.
+
+Here is an example `/etc/pganalyze.env` file to get you started:
 
 ```
 DATABASE_URL=postgres://myusername:mypassword@example.com:5432/mydatabase
-MAILER_URL=smtp://myusername:mypassword@example.com:25
-LICENSE_KEY=KEYKEYKEY
+LICENSE_KEY=REPLACE_ME
 ```
 
-If you'd like to use the Log Insights or Automated EXPLAIN features, or want to run the collector separately from the Enterprise Server container, you also need to pass the [object storage configuration](/docs/enterprise/setup/object-storage) here.
+You can find the full explanation of all variables in the [settings documentation](/docs/enterprise/settings), for example the `MAILER_URL` setting which is required to activate email alerts.
 
-You can find the full explanation of all variables in the [settings documentation](/docs/enterprise/settings).
+### Step 3: Set up object storage
 
+If you'd like to use the Log Insights or Automated EXPLAIN features, or want to run the collector separately from the Enterprise Server container, we now recommend doing the [object storage configuration](/docs/enterprise/setup/object-storage).
 
-### Step 3: Initialize Database & Verify Installation
+### Step 4: Initialize database & verify installation
 
-First, run the Enterprise self check, to ensure the configuration is correct:
+Next, run the Enterprise self check, to ensure the configuration is correct:
```

Comment:
[nitpick] There is no "First" in the new version of doc, yet the following step 5 section also starts with "Next". I guess you can say "Next" followed by "Next", though it might feel a little awkward. Alternatively, we can remove these "Next" and change this sentence to "Run the Enterprise self check to ensure the configuration is correct" (then step 5 will be "Create the initial user").